src/game/client/components/menus.cpp

```diff
@@ -1697,6 +1697,7 @@ int CMenus::Render()
 	else
 	{
 		// render full screen popup
+		char aTitleBuf[128];
```

Comment:
Could also reuse this buffer later on in the method, after the title has been rendered.